keywords/SyncGateway.py

```diff
@@ -156,10 +159,17 @@ def start_sync_gateway(self, cluster_config, url, config):
         log_info("Starting sync_gateway on {} ...".format(target))
         ansible_runner = AnsibleRunner(cluster_config)
         config_path = os.path.abspath(config)
+
+        if is_cbs_ssl_enabled(cluster_config):
+            self.server_port = 18091
+            self.server_scheme = "https"
```

Comment:
The default non-ssl values are initialized when the object is created. The SSL values are set in the start method and can be used in other methods. I thought it is a good idea to initialize the defaults when the object is initialized. Within a test run, the scheme and port won't change.